libr/cons/grep.c

```diff
@@ -278,6 +278,40 @@ R_API void r_cons_grep(const char *str) {
 	}
 }
 
+R_API void r_cons_grep_parsecmd(char *cmd, const char *quotestr) {
+	char *ptr;
+	int len = strlen (cmd);
+
+	//  search and modify '~str~?' into '~?str'
+	if (len > 4 && r_str_endswith (cmd, "~?") && cmd[len - 3] != '\\') {
```

Comment:
this solution only catches this case.. but in fact what ~...~... should concatenate more than one grep after the other. so applying ~str~?  will result in grepping for the lines matching str and then counting them.